pubspec.lock

```diff
@@ -408,6 +415,13 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "1.3.0"
+  uuid:
+    dependency: "direct main"
+    description:
+      name: uuid
+      url: "https://pub.dartlang.org"
+    source: hosted
+    version: "3.0.5"
```

Comment:
Whoopsie 2, I thought it's just the `dropdown_plus` library missing from the lockfile, anyways, we should check in `pubspec.lock` into version control whenever we change `pubspec.yaml`.